core/src/main/java/org/apache/carbondata/core/statusmanager/LoadMetadataDetails.java

```diff
@@ -211,25 +234,32 @@ public long getLoadStartTimeAsLong() {
    * @return
    */
   private long convertTimeStampToLong(String factTimeStamp) {
-    SimpleDateFormat parser = new SimpleDateFormat(CarbonCommonConstants.CARBON_TIMESTAMP_MILLIS);
-    Date dateToStr = null;
     try {
-      dateToStr = parser.parse(factTimeStamp);
-      return dateToStr.getTime();
-    } catch (ParseException e) {
-      LOGGER.error("Cannot convert" + factTimeStamp + " to Time/Long type value" + e.getMessage());
-      parser = new SimpleDateFormat(CarbonCommonConstants.CARBON_TIMESTAMP);
+      // for new loads the factTimeStamp will be long string
```

Comment:
Do not mention "new loads" in line 275. Add comment in line 279, mentioning it is the processing for existing table before carbon 1.3

Reply:
Fixed